File: src/org/ll6/utils/catdisk/entities/Disk.java
package org.ll6.utils.catdisk.entities;

import java.sql.Timestamp;
import java.util.Date;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table (name="TBLDISKS")
public class Disk {

	@Id
	@GeneratedValue(strategy=GenerationType.IDENTITY)
	@Column(name="ID")
	private long diskID;
	
	@Column(name="VolumeName")
	private String volumeName;
	

	@Column(name="DiskSerialNum")
	private String diskSerialNum;
	
	@Column(name="DiskCopyNum")
	private int diskCopyNum;
	
	@Column(name="DiskDate")	
	private Date diskDate;
	
	@Column(name="DescriptionTxt")	
	private String descriptionTxt;
	
	@Column(name="LocationTxt")	
	private String locationTxt;
	
	@Column(name="CreatedOn")	
	private Timestamp createdOn;
	
	@Column(name="ModifiedOn")	
	private Timestamp modifiedOn;

	@Override
	public String toString() {
		return "Disk [diskID=" + diskID + ", volumeName=" + volumeName + ", diskSerialNum=" + diskSerialNum
				+ ", diskCopyNum=" + diskCopyNum + ", diskDate=" + diskDate + ", descriptionTxt=" + descriptionTxt
				+ ", locationTxt=" + locationTxt + ", createdOn=" + createdOn + ", modifiedOn=" + modifiedOn + "]";
	}

	public long getDiskID() {
		return diskID;
	}

	public void setDiskID(long diskID) {
		this.diskID = diskID;
	}

	public String getVolumeName() {
		return volumeName;
	}

	public void setVolumeName(String volumeName) {
		this.volumeName = volumeName;
	}

	public String getDiskSerialNum() {
		return diskSerialNum;
	}

	public void setDiskSerialNum(String diskSerialNum) {
		this.diskSerialNum = diskSerialNum;
	}

	public int getDiskCopyNum() {
		return diskCopyNum;
	}

	public void setDiskCopyNum(int diskCopyNum) {
		this.diskCopyNum = diskCopyNum;
	}

	public Date getDiskDate() {
		return diskDate;
	}

	public void setDiskDate(Date diskDate) {
		this.diskDate = diskDate;
	}

	public String getDescriptionTxt() {
		return descriptionTxt;
	}

	public void setDescriptionTxt(String descriptionTxt) {
		this.descriptionTxt = descriptionTxt;
	}

	public String getLocationTxt() {
		return locationTxt;
	}

	public void setLocationTxt(String locationTxt) {
		this.locationTxt = locationTxt;
	}

	public Timestamp getCreatedOn() {
		return createdOn;
	}

	public void setCreatedOn(Timestamp createdOn) {
		this.createdOn = createdOn;
	}

	public Timestamp getModifiedOn() {
		return modifiedOn;
	}

	public void setModifiedOn(Timestamp modifiedOn) {
		this.modifiedOn = modifiedOn;
	}
	
	@Override
	public int hashCode() {
		final int prime = 31;
		int result = 1;
		result = prime * result + ((createdOn == null) ? 0 : createdOn.hashCode());
		result = prime * result + ((descriptionTxt == null) ? 0 : descriptionTxt.hashCode());
		result = prime * result + diskCopyNum;
		result = prime * result + ((diskDate == null) ? 0 : diskDate.hashCode());
		result = prime * result + (int) (diskID ^ (diskID >>> 32));
		result = prime * result + ((diskSerialNum == null) ? 0 : diskSerialNum.hashCode());
		result = prime * result + ((locationTxt == null) ? 0 : locationTxt.hashCode());
		result = prime * result + ((modifiedOn == null) ? 0 : modifiedOn.hashCode());
		result = prime * result + ((volumeName == null) ? 0 : volumeName.hashCode());
		return result;
	}

	@Override
	public boolean equals(Object obj) {
		if (this == obj)
			return true;
		if (obj == null)
			return false;
		if (getClass() != obj.getClass())
			return false;
		Disk other = (Disk) obj;
		if (createdOn == null) {
			if (other.createdOn != null)
				return false;
		} else if (!createdOn.equals(other.createdOn))
			return false;
		if (descriptionTxt == null) {
			if (other.descriptionTxt != null)
				return false;
		} else if (!descriptionTxt.equals(other.descriptionTxt))
			return false;
		if (diskCopyNum != other.diskCopyNum)
			return false;
		if (diskDate == null) {
			if (other.diskDate != null)
				return false;
		} else if (!diskDate.equals(other.diskDate))
			return false;
		if (diskID != other.diskID)
			return false;
		if (diskSerialNum == null) {
			if (other.diskSerialNum != null)
				return false;
		} else if (!diskSerialNum.equals(other.diskSerialNum))
			return false;
		if (locationTxt == null) {
			if (other.locationTxt != null)
				return false;
		} else if (!locationTxt.equals(other.locationTxt))
			return false;
		if (modifiedOn == null) {
			if (other.modifiedOn != null)
				return false;
		} else if (!modifiedOn.equals(other.modifiedOn))
			return false;
		if (volumeName == null) {
			if (other.volumeName != null)
				return false;
		} else if (!volumeName.equals(other.volumeName))
			return false;
		return true;
	}

}

File: php/src/AppBundle/Entity/Disk.php
<?php

namespace AppBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Disk
 *
 * @ORM\Table(name="disk")
 * @ORM\Entity(repositoryClass="AppBundle\Repository\DiskRepository")
 */
class Disk
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="VolumeName", type="string", length=32)
     */
    private $volumeName;

    /**
     * @var string
     *
     * @ORM\Column(name="DiskSerialNum", type="string", length=16, unique=true)
     */
    private $diskSerialNum;

    /**
     * @var int
     *
     * @ORM\Column(name="DiskCopyNum", type="integer")
     */
    private $diskCopyNum;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="DiskDate", type="datetime")
     */
    private $diskDate;

    /**
     * @var string
     *
     * @ORM\Column(name="DescriptionTxt", type="string", length=100, nullable=true)
     */
    private $descriptionTxt;

    /**
     * @var string
     *
     * @ORM\Column(name="LocationTxt", type="string", length=20, nullable=true)
     */
    private $locationTxt;
    /**
     * @var integer
     *
     * @ORM\Column(name="NumberFiles", type="string", length=20, nullable=true)
     */
    private $numberFiles;
    
    /**
     * @var \DateTime
     *
     * @ORM\Column(name="CreatedOn", type="datetime")
     */
    private $createdOn;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="ModifiedOn", type="datetime")
     */
    private $modifiedOn;


    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set volumeName
     *
     * @param string $volumeName
     *
     * @return Disk
     */
    public function setVolumeName($volumeName)
    {
        $this->volumeName = $volumeName;

        return $this;
    }

    /**
     * Get volumeName
     *
     * @return string
     */
    public function getVolumeName()
    {
        return $this->volumeName;
    }

    /**
     * Set diskSerialNum
     *
     * @param string $diskSerialNum
     *
     * @return Disk
     */
    public function setDiskSerialNum($diskSerialNum)
    {
        $this->diskSerialNum = $diskSerialNum;

        return $this;
    }

    /**
     * Get diskSerialNum
     *
     * @return string
     */
    public function getDiskSerialNum()
    {
        return $this->diskSerialNum;
    }

    /**
     * Set diskCopyNum
     *
     * @param integer $diskCopyNum

     * @return Disk
     */
    public function setDiskCopyNum($diskCopyNum)
    {
        $this->diskCopyNum = $diskCopyNum;

        return $this;
    }

    /**
     * Get diskCopyNum
     *
     * @return int
     */
    public function getDiskCopyNum()
    {
        return $this->diskCopyNum;
    }

    /**
     * Set diskDate
     *
     * @param integer $diskDate
     *
     * @return Disk
     */
    public function setDiskDate($diskDate)
    {
        $this->diskDate = $diskDate;

        return $this;
    }

    /**
     * Get diskDate
     *
     * @return int
     */
    public function getDiskDate()
    {
        return $this->diskDate;
    }

    /**
     * Set descriptionTxt
     *
     * @param string $descriptionTxt
     *
     * @return Disk
     */
    public function setDescriptionTxt($descriptionTxt)
    {
        $this->descriptionTxt = $descriptionTxt;

        return $this;
    }

    /**
     * Get descriptionTxt
     *
     * @return string
     */
    public function getDescriptionTxt()
    {
        return $this->descriptionTxt;
    }

    /**
     * Set locationTxt
     *
     * @param string $locationTxt
     *
     * @return Disk
     */
    public function setLocationTxt($locationTxt)
    {
        $this->locationTxt = $locationTxt;

        return $this;
    }

    /**
     * Get locationTxt
     *
     * @return string
     */
    public function getLocationTxt()
    {
        return $this->locationTxt;
    }

    /**
     * Set NumberFiles
     *
     * @param integer $numberFiles
     *
     * @return Tbldisks
     */
    public function setNumberFiles($numberFiles)
    {
        $this->numberFiles = $numberFiles;

        return $this;
    }

    /**
     * Get numberFiles
     *
     * @return integer
     */
    public function getNumberFiles()
    {
        return $this->numberFiles;
    }
    
    /**
     * Set createdOn
     *
     * @param integer $createdOn
     *
     * @return Disk
     */
    public function setCreatedOn($createdOn)
    {
        $this->createdOn = $createdOn;

        return $this;
    }

    /**
     * Get createdOn
     *
     * @return int
     */
    public function getCreatedOn()
    {
        return $this->createdOn;
    }

    /**
     * Set modifiedOn
     *
     * @param integer $modifiedOn
     *
     * @return Disk
     */
    public function setModifiedOn($modifiedOn)
    {
        $this->modifiedOn = $modifiedOn;

        return $this;
    }

    /**
     * Get modifiedOn
     *
     * @return int
     */
    public function getModifiedOn()
    {
        return $this->modifiedOn;
    }
}

